test/misc.jl

```diff
@@ -8,6 +8,11 @@ function test_labels_codes(mg::MetaGraph)
     for (label_1, label_2) in edge_labels(mg)
         @test has_edge(mg, code_for(mg, label_1), code_for(mg, label_2))
     end
+    # below: arrange(edges) ⊆ keys of mg.edge_data. then = because same length
+    for e in edge_labels(mg)
+        @test_logs mg[e...]  # no log, no error

```

Comment:
What does this mean?

Reply:
It means that this line only tests for the absence of any log record (e.g. no warning via `@warn`) and no Exception / error.

An alternative test could check the output of `arrange` is a key of `edge_data`, but the current test checks the API that users would use rather than the lower-level data structures.

Reply:
sounds good to me